test/stripStyle-test.js

```diff
@@ -1,6 +1,14 @@
 import stripStyle from '../src/stripStyle';
 
 describe('stripStyle', () => {
+  it('should return spring object into value', () => {
+    expect(stripStyle({a: {val: 1, config: [1, 2]}})).toEqual({a: 1});
```

Comment:
can you change these to the opaque `spring(1, [1, 2])`? I'll update the rest of the codebase to follow this.